packages/react-instantsearch-hooks-web/src/widgets/SearchBox.tsx

```diff
@@ -29,33 +29,22 @@ export function SearchBox({ queryHook, ...props }: SearchBoxProps) {
 
   function onReset() {
     setValue('');
+    refine('');
   }
 
   function onChange(event: React.ChangeEvent<HTMLInputElement>) {
-    setValue(event.currentTarget.value);
+    const targetValue = event.target.value;
```

Comment:
we're changing from currentTarget to target, could that have any impact? I don't think so as the listener is directly on the input

Reply:
That wasn't intentional, I'll revert this.